CodeTestHelper.java

```diff
@@ -1333,7 +1337,7 @@ public String getCodeWithoutComments() {
         return code;
     }
 
-    public String removeComments(String code) {
+    public static String removeComments(String code) {
```

Comment:
This wouldn't affect anything in the current tests, would it?

Reply:
It shouldn't. It's only actually used within this class (and probably should be private).